x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -342,12 +342,15 @@ const inProgressTitle = (action: ActionStatus) => (
     defaultMessage="{inProgressText} {nbAgents} {agents} {reassignText}{upgradeText}{failuresText}"
     values={{
       nbAgents:
-        action.nbAgentsAck >= action.nbAgentsActioned
+        action.nbAgentsAck >= action.nbAgentsActionCreated
```

Comment:
Explanation of these changes in [this comment](https://github.com/elastic/kibana/pull/157386#issuecomment-1545782227)

Reply:
`nbAgentsActioned` is the number of total agents that were selected for action (including hosted)
`nbAgentsActionCreated` is the number of agents included in the resulting `.fleet-actions` document (agents array)

I think how this was supposed to work is that we include hosted actions as well in the actions doc, and then write an error to the `.fleet-actions-results` index, so the numbers are adding up, with one failure saying that the hosted agent was not upgradeable.
It looks like on the screenshot this is not what happened, and the hosted agent was excluded from the action (leaving 1 agent in updating state)

Reply:
oh I see! I misunderstood your comment. I'm not sure why that logic didn't work, it surprised me as well.